### DISCUSSION TITLE
How can I freeze the position / rotation of a Dynamic body on a given axis?

### DISCUSSION THREAD

**Discussion by alienself** [jrouwe/JoltPhysics#1086] [General]
Hi there!

Quick question: in PhysX it is possible to freeze the position / rotation of rigid bodies on a given axis, 
for instance freezing the x and z axis which means that the object will only be able to move on the y axis in the physics simulation.

Is there such a concept in Jolt?

**Comment by jrouwe** [comment]
https://github.com/jrouwe/JoltPhysics/blob/2473382d3cae7543840bbfd03b7092071ff96bea/Jolt/Physics/Body/BodyCreationSettings.h#L94

**Comment by PhoenixIllusion** [comment]
During runtime, these can also be changed via a Body's MotionProperties
https://github.com/jrouwe/JoltPhysics/blob/2473382d3cae7543840bbfd03b7092071ff96bea/Jolt/Physics/Body/MotionProperties.h#L92
